shortcodes/class.geojson-shortcode.php

```diff
@@ -156,7 +226,11 @@ function pointToLayer (feature, latlng) {
                     if (circleMarker) {
                         return L.circleMarker(latlng);
                     } else {
-                        return L.marker(latlng);
+                        if(options.iconUrl) {
```

Comment:
I'm guessing this if statement should just always be `return L.marker(latlng, options);` since options defaults to `{}`

Reply:
You're right. I updated, the code is cleaner like this.